src/shared/keyring-util.c

```diff
@@ -5,34 +5,31 @@
 #include "missing_syscall.h"
 
 int keyring_read(key_serial_t serial, void **ret, size_t *ret_size) {
-        size_t m = 100;
+        long bufsize = 100;
```

Comment:
Hmm, why change this to long? We generally want to stick with size_t in our codebase

Reply:
Because `keyctl` expects and returns longs, not `size_t`

https://github.com/systemd/systemd/blob/e1390da0256bbe2017c4c2fbc636c54fe02c84cb/src/basic/missing_syscall.h#L202-L212

https://www.man7.org/linux/man-pages/man2/keyctl.2.html

Reply:
Yes. But as said, we prefer size_t for consistency